cmd/cmd.go

```diff
@@ -180,6 +184,36 @@ func (m *Manager) Run(args []string) {
 	} else if displayVersion {
 		args = []string{"version"}
 	}
+	if len(target) > 0 {
+		var targets = map[string]string{}
```

Comment:
I don't think this logic should be here. Here you could simply call `os.Setenv("TSURU_TARGET", target)` and move this logic to the `ReadTarget()` using the `TSURU_TARGET` env as the `target` variable here.

Also, instead of manually parsing the targets file you can call the `getTargets()` function there which already returns a `map[string]string` for all targets in the targets file.